synthesizer/program/src/logic/instruction/operand/bytes.rs

```diff
@@ -49,6 +51,14 @@ impl<N: Network> ToBytes for Operand<N> {
             Self::Caller => 4u8.write_le(&mut writer),
             Self::BlockHeight => 5u8.write_le(&mut writer),
             Self::NetworkID => 6u8.write_le(&mut writer),
+            Self::Checksum(program_id) => {
+                7u8.write_le(&mut writer)?;
+                program_id.write_le(&mut writer)
```

Comment:
Since `program_id` is an option, you may need to do something like the following:

```
                match fee {
                    None => 0u8.write_le(&mut writer),
                    Some(fee) => {
                        1u8.write_le(&mut writer)?;
                        fee.write_le(&mut writer)
                    }
```

Reply:
Done in the byte utils: https://github.com/ProvableHQ/snarkVM/blob/d03247689d5017b886116ab6044e51234cb6c4ff/utilities/src/bytes.rs#L459

Reply:
It's since been removed from the byte utils and inlined in `operand/bytes.rs`